assets/large_language_models/rag/environments/rag_embeddings/context/conda_dependencies.yaml

```diff
@@ -16,6 +16,6 @@ dependencies:
   - azure-keyvault-secrets==4.6.0
     # package dependency needed for aml run token authentication
   - python-dateutil>=2.7.3,<3.0.0
-  - cryptography!=1.9,!=2.0.*,!=2.1.*,!=2.2.*,<41
+  - cryptography>=41.0.0
```

Comment:
I don't think you need cryptography here, the old line seems like copied from azureml-core setup.py